cmd/pctl/install.go

```diff
@@ -67,7 +68,17 @@ func installCmd() *cli.Command {
 			&cli.StringFlag{
 				Name:  "repo",
 				Value: "",
-				Usage: "The repository to open a pr against. Format is: org/repo-name",
+				Usage: "The repository to open a pr against. Format is: org/repo-name.",
```

Comment:
shall we update the `UsageText` above to demo the two alternative ways of installing a profile?

Reply:
done